code/game/objects/structures/lavaland/geyser.dm

```diff
@@ -44,7 +44,7 @@
 
 /obj/structure/geyser/random
 	erupting_state = null
-	var/list/options = list(/datum/reagent/oil = 2, /datum/reagent/clf3 = 1) //fucking add more
+	var/list/options = list(/datum/reagent/oil = 2, /datum/reagent/clf3 = 1, /datum/reagent/plasma_oxide = 8) //fucking add more //austation -- RDX, TATP
```

Comment:
plasma oxide*

Reply:
yes, it’s actually named that

Reply:
I’m talking about the code comment silly

Reply:
It's referencing the pr that added RDX and tatp, but whatever